### PR TITLE
Isostatic rebound interactive addition

### PR DESCRIPTION
**Pull request by zmustapha**

### CODE REVIEW

File: media/js/isostaticrebound.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    jQuery("#base-content").fadeOut(0, function() {
+        jQuery.mobile.showPageLoadingMsg();                                            
+        jQuery("div.pageview").removeClass(gOrientation).addClass(orientation);
+        jQuery("div.ui-content").removeClass(gOrientation).addClass(orientation);
+        gOrientation = orientation;
+        jQuery("#base-content").fadeIn(function() {
+            jQuery.mobile.hidePageLoadingMsg();    
+        });                        
+    });
+    if (orientation == 'landscape') {
```
Comment (sdreher) [inline media/js/isostaticrebound.js]: should use the === operator

File: media/js/isostaticrebound.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+var gOrientation = undefined;
+
+function orientationLabel() {                    
+    var rotation = window.hasOwnProperty("orientation") ?
+        window.orientation : 90;
+    
+    return (rotation == 0 || rotation == 180) ?
```
Comment (sdreher) [inline media/js/isostaticrebound.js]: swap == with ===

File: polarexplorer/templates/main/isostatic_rebound.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    <!-- Mobile IE allows us to activate ClearType technology for smoothing fonts for easy reading -->
+    <meta http-equiv="cleartype" content="on">
+    
+    <!-- For mobile browsers that do not recognize the viewport tag -->
+    <meta name="MobileOptimized" content="320" />
+    
+    <link rel="stylesheet" href="{{STATIC_URL}}jquery.mobile/jquery.mobile-1.3.2.min.css" media="screen" />
+    
+    {% compress css %}            
+    {% block css %}
```
Comment (sdreher) [inline polarexplorer/templates/main/isostatic_rebound.html]: the blocks are probably unnecessary here as this template is unlikely to be used as a base.

File: media/js/isostaticrebound.js (hunk 1 above)
Comment (zmustapha) [inline media/js/isostaticrebound.js]: I lifted that off the interactive.js, which is the script for both glacier and sea water rise interactives.

File: media/js/isostaticrebound.js (hunk 2 above)
Comment (zmustapha) [inline media/js/isostaticrebound.js]: I lifted that off the interactive.js, which is the script for both glacier and sea water rise interactives.

File: polarexplorer/templates/main/isostatic_rebound.html [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    {% endcompress %}
+    <script type="text/javascript" charset="utf-8" src="{{STATIC_URL}}final.hyperesources/final_hype_generated_script.js?36743"></script>
+
+    <script>
+      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    
+      ga('create', 'UA-311226-46', 'columbia.edu');
```
Comment (sdreher) [inline polarexplorer/templates/main/isostatic_rebound.html]: Take a look at the django-ga-context to replace hard-coded analytics IDs in the repo.

File: media/js/isostaticrebound.js (hunk 1 above)
Comment (zmustapha) [inline media/js/isostaticrebound.js]: See https://github.com/ccnmtl/polarexplorer/blob/master/media/js/interactive.js#L99

File: media/js/isostaticrebound.js (hunk 2 above)
Comment (zmustapha) [inline media/js/isostaticrebound.js]: See https://github.com/ccnmtl/polarexplorer/blob/master/media/js/interactive.js#L99

File: media/js/isostaticrebound.js (hunk 1 above)
Comment (sdreher) [inline media/js/isostaticrebound.js]: Sure, those were created before our more strict jshint adherence.

File: polarexplorer/templates/main/isostatic_rebound.html (hunk 4 above)
Comment (nikolas) [inline polarexplorer/templates/main/isostatic_rebound.html]: I see the django-ga-context tarball on jasper, but I don't see it on pypi or on github/our internal git server. @thraxil could you put this package up on github/pypi?

Comment (zmustapha) [inline polarexplorer/templates/main/isostatic_rebound.html]: The django-ga-context is on in the requirements.txt for polarexplorer. How/what should I be doing for this?

Comment (sdreher) [inline polarexplorer/templates/main/isostatic_rebound.html]: Check the implementation here: https://github.com/ccnmtl/meaningfulconsent/blob/master/meaningfulconsent/templates/base.html#L284

Then, your GA_CODE is added to the salt_state app's local_settings. (Happy to help with that part.)

**Pull request merged**